innovation.game.php

```diff
@@ -7300,6 +7300,22 @@ function stPlayerInvolvedTurn() {
                 self::setGameStateValue('card_id_3', self::executeDraw($player_id, $top_green_card_age, 'hand')['id']);
                 $step_max = 2; // --> 2 interactions: see B
                 break;
+				
+            // id 117, Artifacts age 1: Electrum Stater of Efesos
+            case "117N1":
+                while(true) {
+                    $card = self::executeDraw($player_id, 3, 'revealed'); // "Draw and reveal a 1"
```

Comment:
The comment should read: "Draw and reveal a 3"

Reply:
done